pandas/core/config_init.py

```diff
@@ -239,10 +239,16 @@
 
 
 def mpl_style_cb(key):
+    import warnings
     import sys
     from pandas.tools.plotting import mpl_stylesheet
     global style_backup
 
+    msg = """
```

Comment:
That's what I did initially but the warning was being printed twice. I think since `mpl_style` has a callback `mpl_style_cb` the `_warn_if_deprecated` was being called twice. I'll see what I can do about that tonight.